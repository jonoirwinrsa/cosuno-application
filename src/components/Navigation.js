import { Layout, Menu } from "antd";
import React from "react";
import { Link } from "react-router-dom";

const Navigation = () => (
  <Layout.Header>
    <Menu mode="horizontal" theme="dark" style={{ lineHeight: "64px" }}>
      <Menu.Item>
        <Link to="/">Home</Link>
      </Menu.Item>
      <Menu.Item>
        <Link to="/companies">Companies</Link>
      </Menu.Item>
    </Menu>
  </Layout.Header>
);

export default Navigation;
