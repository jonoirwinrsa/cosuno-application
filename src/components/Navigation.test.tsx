import { cleanup } from "@testing-library/react";
import React from "react";
import { renderWithRouter } from "../testUtils";
import Navigation from "./Navigation";

afterEach(cleanup);

// we are already testing navigation in App.test so let's just check this matches our snapshot
test("<Navigation /> should render correctly", () => {
  const { container } = renderWithRouter(<Navigation />);

  expect(container.firstChild).toMatchSnapshot();
});
