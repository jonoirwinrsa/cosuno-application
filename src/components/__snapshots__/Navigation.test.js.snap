// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> should render correctly 1`] = `
<header
  class="ant-layout-header"
>
  <ul
    class="ant-menu ant-menu-dark ant-menu-root ant-menu-horizontal"
    role="menu"
    style="line-height: 64px;"
  >
    <li
      class="ant-menu-submenu ant-menu-submenu-horizontal ant-menu-overflowed-submenu"
      role="menuitem"
      style="display: none;"
    >
      <div
        aria-expanded="false"
        aria-haspopup="true"
        class="ant-menu-submenu-title"
      >
        <span>
          ···
        </span>
        <i
          class="ant-menu-submenu-arrow"
        />
      </div>
    </li>
    <li
      class="ant-menu-item"
      role="menuitem"
    >
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li
      class="ant-menu-submenu ant-menu-submenu-horizontal ant-menu-overflowed-submenu"
      role="menuitem"
      style="display: none;"
    >
      <div
        aria-expanded="false"
        aria-haspopup="true"
        class="ant-menu-submenu-title"
      >
        <span>
          ···
        </span>
        <i
          class="ant-menu-submenu-arrow"
        />
      </div>
    </li>
    <li
      class="ant-menu-item"
      role="menuitem"
    >
      <a
        href="/companies"
      >
        Companies
      </a>
    </li>
    <li
      class="ant-menu-submenu ant-menu-submenu-horizontal ant-menu-overflowed-submenu"
      role="menuitem"
      style="visibility: hidden; position: absolute; display: none;"
    >
      <div
        aria-expanded="false"
        aria-haspopup="true"
        class="ant-menu-submenu-title"
      >
        <span>
          ···
        </span>
        <i
          class="ant-menu-submenu-arrow"
        />
      </div>
    </li>
  </ul>
</header>
`;
